.github/workflows/release-please.yml

```diff
@@ -8,9 +8,9 @@ jobs:
   release-please:
     runs-on: ubuntu-latest
     steps:
-      - uses: GoogleCloudPlatform/release-please-action@v2
+      - uses: googleapis/release-please-action@v4
```

Comment:
`GoogleCloudPlatform/release-please-action@v2` is outdated. And we need newly to release tags with directory prefix such as `dockertestx/v0.1.7`